pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "humanize",
     "packaging",
     "diskcache",
+    "tqdm",
```

Comment:
added tqdm for progress bars in downloads. we'll use this elsewhere too in later prs